src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -39,7 +39,9 @@ class NUnit extends ConventionTask {
 
     File getNunitExec() {
         assert getNunitHome(), "You must install NUnit and set nunit.home property or NUNIT_HOME env variable"
-        File nunitExec = nunitBinFile("nunit-console${useX86 ? '-x86' : ''}.exe")
```

Comment:
x86 seems to be a console param instead on NUnit v3 (as they're using a separate daemon to actually run the tests)